exercises/none_pattern_test.rb

```diff
@@ -17,37 +17,44 @@ def test_double_negative
     numbers = [9, 3, 3, 7, 6, -5, 1]
     not_none_negative = true
     numbers.each do |number|
-      # Your code goes here
+      not_none_negative = false if number < 0
     end
     refute not_none_negative
   end
 
   def test_none_are_negative
-    skip
     numbers = [9, 3, 1, 8, 3, 3, 5]
     none_negative = true
-    # Your code goes here
+    numbers.each do |num|
+      none_negative = false if num < 0
+    end
     assert none_negative
   end
 
   def test_none_shall_pass
-    skip
     critters = ["elf", "hobbit", "dwarf", "wizard", "human"]
-    # Your code goes here
+    none_shall_pass = true
+    critters.each do |critter|
+      none_shall_pass = false if critter == 'pass'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.